src/ast.rs

```diff
@@ -4044,6 +4044,7 @@ define_vocabulary! {
         DecimalFloat16   (b"Dh", "half"),
         Char32           (b"Di", "char32_t"),
         Char16           (b"Ds", "char16_t"),
+        Char8            (b"Du", "char8_t"),
```

Comment:
Please add this to the comment above too.

Reply:
done